app/src/main/java/org/fossasia/openevent/general/data/Network.kt

```diff
@@ -11,7 +11,7 @@ class Network {
     }
 
     fun isNetworkConnected(): Boolean {
-        val connectivityManager = context.getSystemService(Context.CONNECTIVITY_SERVICE) as? ConnectivityManager
+        val connectivityManager by lazy { context.getSystemService(Context.CONNECTIVITY_SERVICE) as? ConnectivityManager }
```

Comment:
No use unless you make it a global variable

Reply:
Updated